### PR TITLE
Edit Readme

### PR DESCRIPTION
**Pull request by abhinav3967**
The Readme was kind of empty so i just wrote some info

### CODE REVIEW

**Comment by Technohacker** [comment]
I reworded some of the sections, can you have a look into my changes to make sure I haven't missed any details?

**Comment by abhinav3967** [comment]
As of now you can write instruments only in gdscript

**Comment by Technohacker** [comment]
Changed, any other details?

**Comment by abhinav3967** [comment]
lemme see

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 What is GoDAW
 -------------
-GoDAW is a free and open source [DAW](https://en.wikipedia.org/wiki/Digital_audio_workstation) made using the [Godot Game Engine](https://godotengine.org/).
+GoDAW is a free and open source [DAW](https://en.wikipedia.org/wiki/Digital_audio_workstation) made using the [Godot](https://godotengine.org/) Game Engine.
 
 > Note: GoDAW is still in it's early alpha stage.
 
-Why GoDAW ?
+Why a new DAW ?
```
Comment (Technohacker) [inline README.md:9]: Maybe a plain "Why?" will do :P

**Pull request merged**